install-dependencies.sh

```diff
@@ -9,6 +9,5 @@ npm install -g gitbook-cli
 # For generating course website with Jekyll
 gem install bundler
 
-# For converting remark presentations to pdf
-
+# For converting remark presentations to PDF
```

Comment:
But now some `pdf` are lower and some are `PDF` are upper in comments

Reply:
In this file they where all CAPS, and by ALL i mean the other ONE.

Reply:
In the dependencies file we have referred to it as .pdf